lib/segment/src/segment.rs

```diff
@@ -1325,6 +1325,7 @@ impl SegmentEntry for Segment {
     ) -> OperationResult<bool> {
         self.handle_version_and_failure(op_num, None, |segment| match field_type {
             Some(schema) => {
+                // todo(ivan)
```

Comment:
todo what?

Reply:
Sorry, it was obsolete todo that I already done, removed